.mlc-config.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Is this file needed?

Reply:
yes, but only for 1 link that is incorrectly flagged as broken. I minimized the file and moved it out of the root.